pkg/resource/scaling_policy/manager_test_suite_test.go

```diff
@@ -100,7 +101,9 @@ func (d *testRunnerDelegate) Equal(a acktypes.AWSResource, b acktypes.AWSResourc
 	ac := a.(*resource)
 	bc := b.(*resource)
 	// Ignore LastTransitionTime since it gets updated each run.
-	opts := []cmp.Option{cmpopts.EquateEmpty(), cmpopts.IgnoreFields(ackv1alpha1.Condition{}, "LastTransitionTime")}
+	opts := []cmp.Option{cmpopts.EquateEmpty(), cmpopts.IgnoreFields(ackv1alpha1.Condition{}, "LastTransitionTime"),
```

Comment:
How come we're still using go-cmp?

Reply:
Ryan is working on a separate PR to fix all of these issues.